src/Mvc/Mvc.Core/src/Microsoft.AspNetCore.Mvc.Core.csproj

```diff
@@ -48,6 +48,7 @@ Microsoft.AspNetCore.Mvc.RouteAttribute</Description>
     <Reference Include="Microsoft.AspNetCore.Http" />
     <Reference Include="Microsoft.AspNetCore.Http.Extensions" />
     <Reference Include="Microsoft.AspNetCore.ResponseCaching.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.OutputCaching" />
```

Comment:
Thought we normally only referenced `{blah}.Abstractions` from Mvc.Core for middlewares. Why doesn't Microsoft.AspNetCore.OutputCaching.Abstractions exist❔

Reply:
Abstractions were removed (after api review) since it ships in Framework and an abstraction is not on nuget.